docs/asciidoc/back_matter.adoc

```diff
@@ -39,3 +37,94 @@ This spec defines a standard **Data Repository Service (DRS) API** (“the yello
+
+image::prefix_register_1.png[]
+
+image::prefix_register_2.png[]
+
+=== Calling Meta-Resolver APIs for Compact Identifier-Based DRS URIs
+
+Clients resolving Compact Identifier-based URIs need to convert a prefix (e.g. “drs.42”) into an URL pattern. They can do so by calling either the identifiers.org or the n2t.net API, since the two meta-resolvers keep their mapping databases in sync.
+
+=== Calling the identifiers.org API as a Client
+
+It takes two API calls to get the URL pattern.
+
+(i) The client makes a GET request to identifiers.org to find information about the prefix:
+
+  GET https://registry.api.identifiers.org/restApi/namespaces/search/findByPrefix?prefix=drs.42
+
+This request returns a JSON structure including various URLs containing an embedded namespace id, such as:
+
+  "namespace" : {
+   "href":"https://registry.api.identifiers.org/restApi/namespaces/1234"
+  }
+
+(ii) The client extracts the namespace id (in this example 1234), and uses it to make a second GET request to identifiers.org to find information about the namespace:
+
+  GET https://registry.api.identifiers.org/restApi/resources/search/findAllByNamespaceId?id=1234
+
+This request returns a JSON structure including an urlPattern field, whose value is an URL pattern containing a `${id}` parameter, such as:
+
+  "urlPattern" : "https://drs.myexample.org/ga4gh/drs/v1/objects/{$id}"
+
+=== Calling the n2t.net API as a Client
+
+It takes one API call to get the URL pattern.
+
+The client makes a GET request to n2t.net to find information about the namespace. (Note the trailing colon.)
+
+  GET https://n2t.net/drs.42:
+
+This request returns a text structure including a redirect field, whose value is an URL pattern containing a `$id` parameter, such as:
+
+   redirect: https://drs.myexample.org/ga4gh/drs/v1/objects/$id
+
+=== Caching with Compact Identifiers
+
+Identifiers.org/n2t.net compact identifier resolver records do not change frequently.  This reality is  useful for caching resolver records and their URL patterns for performance reasons.  Builders of systems that use compact identifier-based DRS URIs should cache prefix resolver records from identifiers.org/n2t.net and occasionally refresh the records (such as every 24 hours).  This approach will reduce the burden on these community services since we anticipate many DRS URIs will be regularly resolved in workflow systems.  The implementation of the caching approach is up to system builders but we recommend at least caching the mapping of prefixes to URL patterns. System builders may also choose to cache the compact identifier accession to final DRS URL records as well, but that decision/design is at the discretion of the system builder.  Alternatively, system builders may decide to directly mirror the registries themselves, instructions are provided on the identifiers.org/n2t.net websites.
```

Comment:
What new information are we providing here, beyond the (important) guidance in the main text that clients SHOULD cache URL patterns? Unless I'm missing something, I suggest deleting this section, since as written it dilutes the main guidance. (I can provide more detailed comments on specific content, if you point out the key new messages you want to convey here.)

Reply:
Ian brought up the importance of the last line here "Alternatively, system builders may decide to directly mirror the registries themselves, instructions are provided on the identifiers.org/n2t.net websites."

I went ahead and streamlined this section to describe caching a bit more but without just restating the line in the main text doc